ehr2meds/data/generation/standard.py

```
@@ -7,6 +7,140 @@
+            linked_file = col_info["file"]
+            linked_on = col_info["linked_on"]
+            rename_to = col_info.get("rename_to")
+            linked_type = col_info["type"]
+
+            # Load the linked DataFrame from CSV file
+            linked_file_path = output_dir / f"{linked_file}.csv"
+            if not linked_file_path.exists():
+                raise ValueError(
+                    f"Linked file '{linked_file_path}' not found. "
+                    f"Make sure it's generated in the 'data' section first."
+                )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)